Version 2.2 fails loading fixtures

After recent update to 2.2 version i've got this:

>
[Hautelook\AliceBundle\Alice\DataFixtures\LoadingLimitException]                            
  Loading files limit of 50 reached. Could not load the following files 
/code/src/AppBundle/DataFixtures/ORM/changelogs.yml:                                        
   - Entity with ID @user-1 and class Mailer\AuthBundle\Entity\User not found

And the same for every fixtures file, where references are present.

Version 2.1.4 works fine. Has anyone faced this problem too?

Can confirm the issue with hautelook/alice-bundle too:

```bash
$ ./docker-console make app_fixtures
./app/console --no-interaction --env=dev hautelook_alice:doctrine:fixtures:load --purge-with-truncate --bundle=AppBundle --manager=default
  > fixtures found:
      - /code/src/AppBundle/DataFixtures/ORM/incident.yml
      - /code/src/AppBundle/DataFixtures/ORM/changelog.yml
      - /code/src/AppBundle/DataFixtures/ORM/ticket.yml
      - /code/src/AppBundle/DataFixtures/ORM/payment_method.yml
      - /code/src/AppBundle/DataFixtures/ORM/puppet.yml
      - /code/src/AppBundle/DataFixtures/ORM/valid_tld.yml
      - /code/src/AppBundle/DataFixtures/ORM/package.yml
      - /code/src/AppBundle/DataFixtures/ORM/credit_card.yml
      - /code/src/AppBundle/DataFixtures/ORM/operation.yml
      - /code/src/AppBundle/DataFixtures/ORM/cron.yml
      - /code/src/AppBundle/DataFixtures/ORM/news.yml
      - /code/src/AppBundle/DataFixtures/ORM/email.yml
      - /code/src/AppBundle/DataFixtures/ORM/group.yml
      - /code/src/AppBundle/DataFixtures/ORM/server.yml
      - /code/src/AppBundle/DataFixtures/ORM/user_browser.yml
      - /code/src/AppBundle/DataFixtures/ORM/organization.yml
      - /code/src/AppBundle/DataFixtures/ORM/accounting.yml
      - /code/src/AppBundle/DataFixtures/ORM/ip_v4.yml
      - /code/src/AppBundle/DataFixtures/ORM/os.yml
      - /code/src/AppBundle/DataFixtures/ORM/purchase.yml
      - /code/src/AppBundle/DataFixtures/ORM/api_key.yml
      - /code/src/AppBundle/DataFixtures/ORM/invoice.yml
      - /code/src/AppBundle/DataFixtures/ORM/alert.yml
      - /code/src/AppBundle/DataFixtures/ORM/bank_account.yml
      - /code/src/AppBundle/DataFixtures/ORM/server_service.yml
      - /code/src/AppBundle/DataFixtures/ORM/service_ip.yml
      - /code/src/AppBundle/DataFixtures/ORM/domain.yml
      - /code/src/AppBundle/DataFixtures/ORM/country.yml
      - /code/src/AppBundle/DataFixtures/ORM/user.yml
      - /code/src/AppBundle/DataFixtures/ORM/backup_model.yml
      - /code/src/AppBundle/DataFixtures/ORM/product.yml
      - /code/src/AppBundle/DataFixtures/ORM/ip_group.yml
      - /code/src/AppBundle/DataFixtures/ORM/ip_v6.yml
  > purging database

                                                                                                                               
  [Hautelook\AliceBundle\Alice\DataFixtures\LoadingLimitException]                                                             
  Loading files limit of 5 reached. Could not load the following files:                                                        
  /code/src/AppBundle/DataFixtures/ORM/cron.yml:                                                                               
   - Instance mask "domain_*" did not match any existing instance, make sure the object is created after its references        
  /code/src/AppBundle/DataFixtures/ORM/domain.yml:                                                                             
   - Instance mask "server_nx102*" did not match any existing instance, make sure the object is created after its references   
  /code/src/AppBundle/DataFixtures/ORM/incident.yml:                                                                           
   - Instance mask "server_*" did not match any existing instance, make sure the object is created after its references        
  /code/src/AppBundle/DataFixtures/ORM/ip_v4.yml:                                                                              
   - Instance server_default is not defined                                                                                    
  /code/src/AppBundle/DataFixtures/ORM/operation.yml:                                                                          
   - Instance cron_98 is not defined                                                                                           
   - Instance cron_95 is not defined                                                                                           
   - Instance cron_6 is not defined                                                                                            
   - Instance cron_99 is not defined                                                                                           
   - Instance cron_49 is not defined                                                                                           
   - Instance cron_41 is not defined                                                                                           
  /code/src/AppBundle/DataFixtures/ORM/package.yml:                                                                            
   - Instance mask "server_*" did not match any existing instance, make sure the object is created after its references        
  /code/src/AppBundle/DataFixtures/ORM/puppet.yml:                                                                             
   - Entity with ID @puppet_package_www-servers_apache and class AppBundle\Entity\PuppetPackage not found                      
  /code/src/AppBundle/DataFixtures/ORM/server.yml:                                                                             
   - Instance os_gentoo is not defined                                                                                         
   - Instance mask "puppet_class_*" did not match any existing instance, make sure the object is created after its references  
  /code/src/AppBundle/DataFixtures/ORM/service_ip.yml:                                                                         
   - Instance mask "server_*" did not match any existing instance, make sure the object is created after its references        
                                                                                                                               

hautelook_alice:doctrine:fixtures:load [-b|--bundle [BUNDLE]] [-em|--manager MANAGER] [--append] [--shard SHARD] [--purge-with-truncate] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-s|--shell] [--process-isolation] [-e|--env ENV] [--no-debug] [--] <command>

Makefile:55: recipe for target 'app_fixtures' failed
make: *** [app_fixtures] Error 1
```

@theofidry Can we have a clue of the reason of this bug? :-)

--- a/src/Nelmio/Alice/Instances/Processor/Methods/Parameterized.php
+++ b/src/Nelmio/Alice/Instances/Processor/Methods/Parameterized.php
@@ -16,7 +16,7 @@ use Nelmio\Alice\Instances\Processor\ProcessableInterface;
 
 class Parameterized implements MethodInterface
 {
-    private static $regex = '/<\{(?<parameter>.*)\}>/i';
+    private static $regex = '/<\{(?<parameter>.*?)\}>/i';
 
     /**
      * @var ParameterBag
@@ -58,7 +58,7 @@ class Parameterized implements MethodInterface
                 if (false === $this->parameters->has($key)) {
                     throw new \UnexpectedValueException(
                         sprintf(
-                            'Parameter "%s" was not found',
+                            'Parameter "%s" was not found.',
                             $key
                         )
                     );

--- a/tests/Nelmio/Alice/Fixtures/Files/parameters/composite.yml
+++ b/tests/Nelmio/Alice/Fixtures/Files/parameters/composite.yml
@@ -10,8 +10,10 @@
 parameters:
     key1: NaN
     key2: Bat
-    composite: '<{key1}> <{key2>!'
+    composite: '<{key1}> <{key2}>!'
 
 Nelmio\Alice\support\models\User:
     user0:
         username: '<{composite}>'
+    user1:
+        username: '<{key1}> <{key2}>!'

--- a/tests/Nelmio/Alice/Fixtures/LoaderTest.php
+++ b/tests/Nelmio/Alice/Fixtures/LoaderTest.php
@@ -1707,30 +1707,40 @@ class LoaderTest extends \PHPUnit_Framework_TestCase
 
     public function testCompositeParametersLoading()
     {
-        $this->markTestSkipped('Parameters cannot be composite yet.');
-        $res = $this->createLoader()->load(__DIR__ . '/Files/parameters/composite.yml');
+        $objects = $this->createLoader()->load(__DIR__ . '/Files/parameters/composite.yml');
 
-        $this->assertCount(1, $res);
+        $this->assertCount(2, $objects);
 
-        $user = $res['user0'];
+        $user = $objects['user0'];
         $this->assertInstanceOf(self::USER, $user);
-        $this->assertEquals('Nan Bat!', $user->username);
+        //$this->assertEquals('NaN Bat!', $user->username); Not supported yet
+        $this->assertEquals('<{key1}> <{key2}>!', $user->username);
+
+        $user = $objects['user1'];
+        $this->assertInstanceOf(self::USER, $user);
+        //$this->assertEquals('NaN Bat!', $user->username); Not supported yet
+        $this->assertEquals('NaN Bat!', $user->username);
     }
 
+    /**
+     * @expectedException \UnexpectedValueException
+     * @expectedExceptionMessage Parameter "username_<current()>" was not found.
+     */
     public function testDynamicParametersLoading()
     {
-        $this->markTestSkipped('Parameters cannot be dynamic yet.');
-        $res = $this->createLoader()->load(__DIR__ . '/Files/parameters/dynamic.yml');
+        $objects = $this->createLoader()->load(__DIR__ . '/Files/parameters/dynamic.yml');
+        $this->fail('Expected exception to be thrown.');
 
-        $this->assertCount(2, $res);
+        // Skipped: not supported yet
+        //$this->assertCount(2, $objects);
 
-        $alice = $res['user_alice'];
-        $this->assertInstanceOf(self::USER, $alice);
-        $this->assertEquals('Alice', $alice->username);
+        //$alice = $objects['user_alice'];
+        //$this->assertInstanceOf(self::USER, $alice);
+        //$this->assertEquals('Alice', $alice->username);
 
-        $bob = $res['user_bob'];
-        $this->assertInstanceOf(self::USER, $bob);
-        $this->assertEquals('Bob', $bob->username);
+        //$bob = $objects['user_bob'];
+        //$this->assertInstanceOf(self::USER, $bob);
+        //$this->assertEquals('Bob', $bob->username);
     }
 
     public function testBackslashes()

--- a/tests/Nelmio/Alice/Instances/Processor/Methods/ParameterizedTest.php
+++ b/tests/Nelmio/Alice/Instances/Processor/Methods/ParameterizedTest.php
@@ -101,6 +101,10 @@ class ParameterizedTest extends \PHPUnit_Framework_TestCase
                 new Processable('<{<{part1}> <{part2}>}>'),
                 true,
             ],
+            'successive' => [
+                new Processable('<{foo}> <{bar}>'),
+                true,
+            ],
             'dynamic' => [
                 new Processable('<{username_<current()>}>'),
                 true,

